### PR TITLE
Feature/DNN-10034 Save Page Template

### PR DESCRIPTION
**Pull request by ricardodiasc**
Adds the buttons "Add Multiple Pages"  and  "Save Page Template" and a extension point for PageSettings.

### CODE REVIEW

File: src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/App.jsx [hunk 1]
```diff
@@ -443,6 +444,28 @@ class App extends Component {
         });
     }
 
+    onSearchBlur(){
```
Comment (amarjit-dhunna) [inline src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/App.jsx]: Where is this method used?

File: src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/App.jsx [hunk 2]
```diff
@@ -443,6 +444,28 @@ class App extends Component {
...
+        const {searchTerm} = this.state;
+        searchTerm ? this.setState({inSearch:true}) : this.setState({inSearch:false});
+    }
+
+
+    onAddMultiplePage() {
```
Comment (amarjit-dhunna) [inline src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/App.jsx]: Multiple pages addition would be handled separately in a different JIRA.

File: src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/App.jsx [hunk 3]
```diff
@@ -661,42 +664,30 @@ class App extends Component {
     }
 
 
+    onSaveMultiplePages(){
```
Comment (amarjit-dhunna) [inline src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/App.jsx]: Not needed

File: src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/PageSettings/PageSettings.jsx [hunk 4]
```diff
@@ -94,8 +94,8 @@ class PageSettings extends Component {
             this.props.onChangeField("hierarchy", parentPageName);
         }
     }
-
-    render() {
+    
+    render_defaultTabsDetails(){
```
Comment (amarjit-dhunna) [inline src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/PageSettings/PageSettings.jsx]: Where is this used?

**Comment by amarjit-dhunna** [comment]
1. Since you have renamed the Additional Panel so remove all the additional panel code. 
2. The add multiple pages is not part of this JIRA and would be done separately in a different JIRA. It is not happening in same way as if now.
3. Do not commit yarn.lock file

**Review by amarjit-dhunna — CHANGES_REQUESTED** [review]

File: src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/App.jsx (hunk 1 above)
Comment (ricardodiasc) [reply src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/App.jsx]: Merge problem. Removing...

File: src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/PageSettings/PageSettings.jsx (hunk 4 above)
Comment (ricardodiasc) [reply src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/PageSettings/PageSettings.jsx]: Just here. I notice this is just a redirect. Changing the implementation of render_defaultTabsDetails() inside render.

**Comment by kanm2015** [comment]
code review done

**Pull request merged**